AppCenterAnalytics/AppCenterAnalytics/Internals/Session/MSACSessionTrackerPrivate.h

```diff
@@ -16,6 +16,11 @@
  */
 @property(nonatomic, getter=isStarted) BOOL started;
 
+/**
+ * Manual session tracking flag.
+ */
+@property BOOL isManualSessionTrackerEnabled;
```

Comment:
` Manual session tracking flag.` -> `Stores the value of whether manual session tracker is enabled, false by default.`